htmd/parameterization/cli.py

```diff
@@ -525,13 +564,23 @@ def main_parameterize(arguments=None):
     mol = _prepare_molecule(args)
 
     # Copy the molecule to preserve initial coordinates
-    orig_coor = mol.coords.copy()
+    initial_coords = mol.coords.copy()
 
     # Select dihedral angles to parameterize
     selected_dihedrals = _select_dihedrals(mol, args)
 
-    # Get a reference calculator
-    qm = _get_reference_calculator(args)
+    # Get a queue
+    queue = _get_queue(args)
+
+    # Get calculators
+    qm = _get_qm_calculator(args, queue)
+    nnp = _get_nnp_calculator(args, queue)
+    if args.nnp:
+        ref = nnp
```

Comment:
Call ref calculator instead (as the functions already call it). Ref is too abstract a variable name I think

Reply:
Renamed!